module/functions/azure/aad/Assert-AzureAdSecurityGroup.ps1

```diff
@@ -22,6 +22,10 @@ The description of the group
 The object IDs of the principals to assign as owners to the group.
 Note, that if the group already exists, we will not attempt to assign the owners (as the principal may not have sufficient privileges)
 
+.PARAMETER StrictMode
+When true, the group's description forms part of the idempotency check.  If the specified description does not match the group's
+definition in AzureAD, then it will be updated to ensure it matches.
```

Comment:
>then it will be updated to ensure it matches.

except for the owners list, right?

Reply:
This is tailored to an expectation of the automation not having full AAD rights.  We expect to have `Group.Create` permissions which only allows owners to be set upon creation.  So this definitely needs a comment to reflect that assumption!

Reply:
Notes added to the doc comments